### PR TITLE
Optimized sgemv and dgemv kernel LSX for LoongArch

### PR DESCRIPTION
**Pull request by ErnstPeng**
Sgemv and dgemv kernel LSX. @XiWeiGu

### CODE REVIEW

File: kernel/loongarch64/sgemv_t_lsx.S [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    LDARG  $r23,  $sp,    0
+    LDARG  $r24,  $sp,    8
+    LDARG  $r25,  $sp,    16
+    LDARG  $r26,  $sp,    32
+    LDARG  $r27,  $sp,    40
+    LDARG  $r28,  $sp,    48
+    LDARG  $r29,  $sp,    56
+    LDARG  $r30,  $sp,    64
+    LD     ALPHA, $sp,    72
+#ifndef __64BIT__
```
Comment (XiWeiGu) [inline kernel/loongarch64/sgemv_t_lsx.S]: Redundant code

Comment (ErnstPeng) [reply kernel/loongarch64/sgemv_t_lsx.S]: OK

**Comment by XiWeiGu** [comment]
LGTM

**Comment by ErnstPeng** [comment]
Code conflicts are resolved.

**Comment by martin-frbg** [comment]
Thank you

**Pull request merged**